lib/http_proxy_override.dart

```diff
@@ -47,8 +94,8 @@ class HttpProxyOverride extends HttpOverrides {
       environment['http_proxy'] = '$host:$port';
       environment['https_proxy'] = '$host:$port';
     } else {
-      environment['http_proxy'] = '$host:8888';
```

Comment:
Good catch, but I'm not sure if it can be set to `null` or not, or just do nothing